src/v2/spec.yaml

```diff
@@ -7707,7 +7707,7 @@ paths:
     post:
       tags:
         - Action
-      summary: Perform actions on behalf of users across different apps
+      summary: Act on behalf of user
```

Comment:
This is a bit confusing because all write APIs are "acting on behalf of user"

The main thing here is that the actions can be on other apps instead of just Farcaster protocol. 

So something like "Actions in other apps" maybe? Open to suggestions

Reply:
Made this comment before seeing the discussion here: https://neynar.slack.com/archives/C0653FSDW9H/p1729834981561469?thread_ts=1729741037.898749&channel=C0653FSDW9H&message_ts=1729834981.561469

Can continue in slack